spec/requestTest.js

```diff
@@ -0,0 +1,26 @@
+const request = require('request');
+const Games = require('../collections/Games');
+
+const joinOrStartGameURL = 'http://127.0.0.1:1234/api/play/';
```

Comment:
change route name to something like
`/api/games/` or something like that
I can't remember what Zach said, but I think he said it should only be nouns for RESTful api endpoints.